examples/step-47/doc/intro.dox

```diff
@@ -334,8 +334,8 @@ where
 \\
 &
  -\sum_{e \in \mathbb{F}} \int_{e}
-  \jump{ \frac{\partial^2 v_h}{\partial \mathbf n^2}}
-  \average{\frac{\partial u_h}{\partial \mathbf n}} \ ds
+  \jump{ \frac{\partial^2 u_h}{\partial \mathbf n^2}}
+  \average{\frac{\partial v_h}{\partial \mathbf n}} \ ds
```

Comment:
Was this just wrong in the documentation before?

We like to state problems with the test function multiplied from the left instead of the right. Would you mind just switching the two terms?

Reply:
I found it's different from the scheme in the paper.
Yes, I can put the test function on the left.